BDArmory/Control/MissileFire.cs

```diff
@@ -7542,10 +7790,11 @@ void BombAimer()
                     drag = ml.vessel.parts.Sum(x => x.dragScalar);
                 }
 
-                dragForce = (0.008f * bombPart.mass) * drag * 0.5f * simSpeedSquared * atmDensity * simVelocity.normalized;
-                simVelocity -= (dragForce / bombPart.mass) * simDeltaTime;
-                //float lift = 0.5f * atmDensity * simSpeedSquared * launcher.liftArea * BDArmorySettings.GLOBAL_LIFT_MULTIPLIER * MissileGuidance.DefaultLiftCurve.Evaluate(1);
-                //simVelocity += VectorUtils.GetUpDirection(currPos) * lift;
+                dragForce = (0.008f * ordinanceMass) * drag * 0.5f * simSpeedSquared * atmDensity * bombPredictionFudgeFactor * simVelocity.normalized;
```

Comment:
`bombPredictionFudgeFactor` shouldn't be negative, otherwise this is increasing `simVelocity` instead of decreasing it.

It looks like the bomb aimer is using the second variant of semi-implicit Euler (https://en.wikipedia.org/wiki/Semi-implicit_Euler_method#The_method), which ought to be sufficiently accurate, but you could try switching to the first variant where `currPos` is updated after `simVelocity` instead of before, which I think is more inline with what KSP uses in the flight integrator, though this should only make a minor difference.
Also, the `simVelocity` used in the drag calculation should be the velocity prior to it being updated by gravity to be consistent with semi-implicit Euler, e.g.,
```
acceleration = gravity + thrust + drag (at current position and velocity)
velocity += Δt * acceleration
position += Δt * velocity
```
though even then, it's not technically consistent since the acceleration isn't independent of the velocity.
If the error in the aiming position is coming from the numerical integrator, we could try replacing it with the midpoint method (https://en.wikipedia.org/wiki/Midpoint_method, either the explicit or implicit variants), which should be significantly more accurate, but requires more computation. (Leap-frog also technically requires the Hamiltonian to have the `H = T(t,v) + V(t,x)` form, so it wouldn't be any better than semi-implicit Euler here.)
However, that all depends on whether the error in the trajectory is coming from the integrator or from some incorrect initial position/velocity.

Reply:
bombPredictionFudge factor being negative was the result of empirical testing and adjusting the factor until the reticle lined up with the actual impact site; increasing vel was the needed outcome - the bomb aimer was predicting a hit position several hundred meters in front of a target; increasing vel pushes it back until prediction and actual line up. 

Further experimentation will be needed to determine where/what of the prediction calc is incorrectly configured.

Reply:
Taking a look and comparing against other SIE trajectory calculations in ModuleWeapon, found what was going on - The gravity component was getting incorrectly applied twice/wasn't getting properly cancelled out during the leapfrog, resulting in the prediction hitting the ground sooner than it should. Quick test has things now working correctly.